<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CatalogRobot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="6" column="15">
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="download_icon">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/res/download.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2" colspan="16">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>570</width>
             <height>259</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="scrollArea_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="6" column="14">
         <widget class="QPushButton" name="pause_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Пауза</string>
          </property>
         </widget>
        </item>
        <item row="2" column="15">
         <widget class="QPushButton" name="parser_search_button">
          <property name="text">
           <string>Обзор...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="ID">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>42</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="upload_icon">
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/res/upload.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="parser_label">
          <property name="text">
           <string>Парсер</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="password_label">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="login_label">
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="14">
         <widget class="QLineEdit" name="login_lineEdit"/>
        </item>
        <item row="1" column="2" colspan="14">
         <widget class="QLineEdit" name="password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="13">
         <widget class="QLineEdit" name="parser_lineEdit"/>
        </item>
        <item row="3" column="3" colspan="12">
         <widget class="QLabel" name="name">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Название</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QPushButton" name="add_button">
          <property name="text">
           <string>Добавить закупку</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="pause_lineEdit">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string>0000000000</string>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4" colspan="3">
         <widget class="QLabel" name="pause_label">
          <property name="text">
           <string>Пауза, мс</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Посмотреть справку</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Открыть файл настроек</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>login_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>parser_lineEdit</tabstop>
  <tabstop>parser_search_button</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>pause_button</tabstop>
  <tabstop>start_button</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
